### PR TITLE
feat(route): EU Disinfo Lab publications

### LINKED ISSUE #7281: 欧盟虚假信息实验室
### 网站地址
https://www.disinfo.eu/publications
### 需要生成什么内容？
发布的博客文章和刊物
### 额外描述
[死人也能编黑料、空壳智库遍地跑，全球反华水军之王竟然是......](https://mp.weixin.qq.com/s/lvuAxJSFRjFWAuk4PyRSmQ)
[印度编年史: 深入研究一项为期15年的针对欧盟和联合国为印度利益服务的行动Indian Chronicles: deep dive into a 15-year operation targeting the EU and UN to serve Indian interests | EU DisinfoLab](https://www.disinfo.eu/publications/indian-chronicles-deep-dive-into-a-15-year-operation-targeting-the-eu-and-un-to-serve-indian-interests/)

### PR DESCRIPTION
**Pull request by nczitzk**
## 该 PR 相关 Issue / Involved issue

Close #7281

## 完整路由地址 / Example for the proposed route(s)

```routes
/disinfo/publications
```

## 新RSS检查列表 / New RSS Script Checklist

- [x] 这个PR中包含了新的路由吗? Does this PR add new route?
  - 如果有, 请完成检查列表. If yes, please finish the check list
  - **如果你的PR符合下方某个事项, 也请注明. If any of the checklist item meets your PR, please fill it out.**
  - [x] <- 这样打勾
- [x] 是否提供了文档? Documentation provided?
  - [x] 是否提供了英文文档? EN Documentation provided?
- [x] 是否支持全文获取? Is this RSS Script support fulltext?
  - [x] 如果全文获取中需要访问文章链接, 是否使用了缓存? If fulltext requires to fetch detail pages, is cache used in the process?
  - [缓存说明](https://docs.rsshub.app/joinus/#ti-jiao-xin-de-rsshub-gui-ze-bian-xie-jiao-ben-shi-yong-huan-cun) | [How to use cache](https://docs.rsshub.app/joinus/#ti-jiao-xin-de-rsshub-gui-ze-bian-xie-jiao-ben-shi-yong-huan-cun)
- [ ] 目标是否有明显的反爬/频率限制? Is there any sign of anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? (延长缓存时间, 写文档说明, etc.) If yes, do your code reflect this sign? (e.g. write documentations, use long cache time)
- [ ] 是否引入的新的包? Any new package introduced?
  - 如果有, 请说明原因. If yes, please state your reason
- [ ] 是否使用了`Puppeteer`? Make use of `Puppeteer`?
  - 如果有, 请说明原因. If yes, please state your reason

### CODE REVIEW

File: lib/routes/disinfo/publications.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const date = require('@/utils/date');
```
Comment (NeverBehave) [inline lib/routes/disinfo/publications.js:3]: https://docs.rsshub.app/joinus/pub-date.html

**Review by NeverBehave — CHANGES_REQUESTED** [review]

File: lib/routes/disinfo/publications.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    const currentUrl = `${rootUrl}/publications`;
+    const response = await got({
+        method: 'get',
+        url: currentUrl,
+    });
+
+    const $ = cheerio.load(response.data);
+
+    const list = $('.elementor-heading-title a')
+        .slice(0, 10)
```
Comment (SettingDust) [inline lib/routes/disinfo/publications.js:16]: 截断？

**Review by SettingDust — CHANGES_REQUESTED** [review]

File: lib/routes/disinfo/publications.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        list.map(
+            async (item) =>
+                await ctx.cache.tryGet(item.link, async () => {
+                    const detailResponse = await got({
+                        method: 'get',
+                        url: item.link,
+                    });
+                    const content = cheerio.load(detailResponse.data);
+
+                    item.description = content('.elementor-widget-theme-post-content').html();
```
Comment (SettingDust) [inline lib/routes/disinfo/publications.js:35]: 有lazyload 的图

**Review by SettingDust — CHANGES_REQUESTED** [review]

**Review by SettingDust — APPROVED** [review]

**Review by NeverBehave — CHANGES_REQUESTED** [review]
处理一下冲突

**Comment by TonyRL** [comment]
v2 script standard #8236

**Review by TonyRL — CHANGES_REQUESTED** [review]
v2 script standard #8236

**Pull request merged**